src/client/websocket/handlers/CHANNEL_PINS_UPDATE.js

```diff
@@ -6,7 +6,7 @@ module.exports = (client, { d: data }) => {
   const channel = client.channels.get(data.channel_id);
   const time = new Date(data.last_pin_timestamp);
 
-  if (channel && time) {
+  if (channel && time.getTime()) {
```

Comment:
That if has been there since the time it was added. 👏

I think you rather should just remove `time` from it or explicitly handle `NaN` as we want `0` to pass here.
The reason for that is that Discord sends `null` when the last pin was removed, and we still want to emit an update then.

Reply:
Fixed in 36eaa83